packages/twenty-front/src/utils/isTruthy.ts

```diff
@@ -0,0 +1 @@
+export const isTruthy = Boolean;
```

Comment:
We should avoid creating utils like this as it circumvents the rule by applying the same behavior as an if.

Reply:
We want to prevent implicit boolean coercion :  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean#boolean_coercion

Reply:
Removed